### PR TITLE
feat: add CreateOrganizationMutation

### PR DESCRIPTION
**Pull request by JoyLubega**
Refers: opencollective/opencollective/issues/3468

For https://github.com/opencollective/opencollective-frontend/pull/5083

Description: Add website field in the createCollectiveMutation V2

### CODE REVIEW

**Comment by sbinlondon** [comment]
Just leaving a note that I will get to this this morning, apologies for the delay.

**Comment by JoyLubega** [comment]
@sbinlondon Can you help me look at this. When you get time today

**Comment by sbinlondon** [comment]
Hi Joy, this is looking good, does the mutation work well with your frontend work?

**Comment by JoyLubega** [comment]
Yes it works fine both for creating a collective and an organisation

**Comment by sbinlondon** [comment]
Tested it and it works good for me too 👍  Make sure to rebase onto the latest `master` branch.

**Comment by JoyLubega** [comment]
👍

File: server/graphql/v2/object/Collective.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const Collective = new GraphQLObjectType({
   name: 'Collective',
   description: 'This represents a Collective account',
   interfaces: () => [Account, AccountWithHost, AccountWithContributions],
-  isTypeOf: collective => collective.type === collectiveTypes.COLLECTIVE,
+  isTypeOf: collective => collective.type === collectiveTypes.COLLECTIVE || collectiveTypes.ORGANIZATION,
```
Comment (Betree) [inline server/graphql/v2/object/Collective.js:12]: In GraphQL V2, an `Organization` is not a `Collective`, these are two different types. Why is that needed?

File: server/graphql/v2/input/CollectiveCreateInput.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ export const CollectiveCreateInput = new GraphQLInputObjectType({
   fields: () => ({
     name: { type: new GraphQLNonNull(GraphQLString) },
     slug: { type: new GraphQLNonNull(GraphQLString) },
+    website: { type: GraphQLString },
+    type: { type: GraphQLString },
```
Comment (Betree) [inline server/graphql/v2/input/CollectiveCreateInput.js:10]: There should not be a `type` here, if it's a `CollectiveCreateInput` then the type is `Collective`.
If we want to be able to create an organization we need to define a new `OrganizationCreateInput`.

File: server/graphql/v2/object/Collective.js (hunk 1 above)
Comment (JoyLubega) [reply server/graphql/v2/object/Collective.js:12]: Thanks for the clarification, I thought Organizaton is a collective. Let me fix it

File: server/graphql/v2/input/CollectiveCreateInput.js (hunk 2 above)
Comment (JoyLubega) [reply server/graphql/v2/input/CollectiveCreateInput.js:10]: @Betree just to make sure I am doing the right thing, You mean I had to create a mutation in V2 for Organization, and also the file `OrganizationCreateInput`

Comment (JoyLubega) [reply server/graphql/v2/input/CollectiveCreateInput.js:10]: Does it mean I also create new mutations in the frontend.  `CreateOrganizationMutation` and `CreateOrganizationMutation`

Comment (Betree) [reply server/graphql/v2/input/CollectiveCreateInput.js:10]: Yes, you'll need to create new mutations dedicated to organizations in the frontend

**Comment by JoyLubega** [comment]
Hi @Betree @sbinlondon I need some help with writing mutation. I have created new ones both for edit and create organization but I have thins error `Error: Unknown type "OrganizationCreateInput". Did you mean "CollectiveCreateInput", "CommentCreateInput", "CreditCardCreateInput", "ExpenseItemCreateInput", or "OrderCreateInput"?` I have created a new OrganizationInput but as thought its not recognised

**Comment by JoyLubega** [comment]
> Hi @Betree @sbinlondon I need some help with writing mutation. I have created new ones both for edit and create organization but I have thins error `Error: Unknown type "OrganizationCreateInput". Did you mean "CollectiveCreateInput", "CommentCreateInput", "CreditCardCreateInput", "ExpenseItemCreateInput", or "OrderCreateInput"?` I have created a new OrganizationInput but as thought its not recognised

Disregard this. I have fixed it

File: server/graphql/v2/mutation/CreateOrganizationMutation.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    type: activities.COLLECTIVE_CREATED,
+    UserId: remoteUser.id,
+    data: {
+      collective: organization.info,
+      user: {
+        email: remoteUser.email,
+        collective: remoteUserCollective.info,
+      },
+    },
+  });
```
Comment (sbinlondon) [inline server/graphql/v2/mutation/CreateOrganizationMutation.js:59]: For organizations, we don't send out the "collective created" email (see https://github.com/opencollective/opencollective-api/blob/edabab5bcf9dba6418eea2e88be58311a0300361/server/graphql/v1/mutations/collectives.js#L158)

**Review by sbinlondon — CHANGES_REQUESTED** [review]
Looks good to me, just need to remove the lines I commented about 👍

File: server/graphql/v2/mutation/CreateOrganizationMutation.js (hunk 3 above)
Comment (snitin315) [reply server/graphql/v2/mutation/CreateOrganizationMutation.js:59]: @sbinlondon, Here we aren't notifying the `host` but only the `admin`.

Comment (snitin315) [reply server/graphql/v2/mutation/CreateOrganizationMutation.js:59]: I'm removing it as the user is redirected to the newly created org page with a banner at the top saying `Your org is created`

**Review by sbinlondon — APPROVED** [review]
